### ISSUE TITLE
no $data for seeEventTriggered & dontSeeEventTriggered

### ISSUE THREAD

**Issue by dqualmann** [Codeception/module-symfony#170]
Since symfony/http-kernel 6.3.0 `\Symfony\Component\HttpKernel\DataCollector\EventDataCollector` has a new property `$defaultDispatcher`. Now it is no longer possible to use $eventCollector->getCalledListeners, $eventCollector->getNotCalledListeners or $eventCollector->getOrphanedEvents without a filled param `$dispatcher` because `Symfony\Component\HttpKernel\Profiler\Profiler::loadProfileFromResponse` fill not this new property.

Alternative it is possible to use `\Symfony\Component\HttpKernel\DataCollector\EventDataCollector::getData`.

Currently this error occurs: 
```
[Error] Typed property Symfony\Component\HttpKernel\DataCollector\EventDataCollector::$defaultDispatcher must not be accessed before initialization
```

StackTrace:
```
#1 /app/vendor/symfony/http-kernel/DataCollector/EventDataCollector.php:100
#2 Codeception\Module\Symfony->dontSeeEventTriggered
```

**Comment by TavoNiievez** [comment]
Hi @dqualmann ,

A few days ago I released version 3.2.0 of this module and this was one of the things that was corrected.

Could you check again if everything works as expected?

Thank you!

**Comment by dqualmann** [comment]
Hey,

i have been using this version since last Friday. And I was able to remove my quick fix and everything is now running as expected with Symfony Kernel 6.4.

Thanks.
